department-of-veterans-affairs/abd-vro
# Move service-python into domain-rrd

## What was the problem?

`service-python` was its own Gradle project but it contains functionality relating to `domain-rrd`, so it should be contained within that directory

Associated tickets or Slack threads:

- https://github.com/department-of-veterans-affairs/abd-vro/issues/1509

## How does this fix it?[^1]
Moves all code associated with `service-python` into `domain-rrd` and removes references in Gradle project. 

## How to test this PR
- As this is not a logic change, and RRD code is no going to be used anymore, successful build and unit testing is sufficient for testing.


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
